raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2548,6 +2549,10 @@ public OptionalLong highWatermark() {
         }
     }
 
+    public Optional<Node> idToNode(int id, String listener) {
+        return quorum.idToNode(id, listener);
```

Comment:
Let's use `partitionState` directly and remove the changes to `QuorumState`.
```java
        return partitionState.latestVoterSet().idToNode(id, listener);
```

Reply:
ok